src/dotnet-aspnet-codegenerator/DotNetBuildCommandHelper.cs

```diff
@@ -54,6 +54,21 @@ internal class DotNetBuildCommandHelper
                 StdOut = stdOutMsgs
             };
         }
+
+        private static string GetFrameworkShortName(NuGetFramework framework)
+        {
+            string shortFramework = string.Empty;
+            if (framework != null)
+            {
+                if (framework.Framework.Equals(FrameworkConstants.FrameworkIdentifiers.Net) &&
+                    framework.Version.Major >= 5)
+                {
+                    return string.Concat(NetFrameworkShortFolderName, framework.Version.Major, ".", framework.Version.Minor);
+                }
+                shortFramework = framework.GetShortFolderName();
```

Comment:
Why is this not handling the short name in all cases - GetShortFolderName?

Reply:
For .NETFrameworks, NuGet.Frameworks.NuGetFramework.GetShortFolderName returns the version without dots. for example, NET 4.6.1 => net461 and NET 5.0 => net50. For NETCoreApp it returns the version with dots for example NET Core APp 3.1 => netcoreapp3.1. Since the short folder name for NET 5.0 should be net5.0 instead of net50, I'm handling it myself. I might not be consuming the latest Nuget.Frameworks though but can look into that.

Reply:
NET 5.0 should be returned under NETCoreApp. Is that not what you are seeing?

Reply:
NET 5.0 does not return under NETCoreApp since 5.0-Preview5